src/geo/transform.js

```diff
@@ -202,6 +200,25 @@ class Transform {
         this._calcMatrices();
     }
 
+    getProjection() {
+        const {center, parallels} = this.projection;
+
+        // if the user hasn't defined a center and parallels,
+        // use the default ones (if any).
+        return {
+            ...{center},
+            ...(parallels) && {parallels},
+            ...this._projectionOptions
+        };
```

Comment:
Maybe this could be shortened by using a `pick` utility from `util.js`?

```suggestion
        return pick(this.projection, ['name', 'center', 'parallels']);
```

Reply:
I wasn't aware of `pick` but if the center or parallels is set in `_projectionOptions`, we want to use that instead of the default that's in `projection`. I also figured that if `parallels` wasn't set anywhere, we wouldn't necessarily want to return `{..., parallels: undefined, ...}`. `center` and `name` would always be set though.

Reply:
@ryanhamley `this.projection` comes from `getProjection` which already handles overwriting defaults with supplied options, so we can use it. Also, `pick` won't add a key if it's not set in the given source, so it should be fine.